packages/pwa-kit-react-sdk/src/ssr/server/express.test.js

```diff
@@ -540,6 +544,34 @@ describe('SSRServer operation', () => {
             })
     })
 
+    describe('Running remotely', () => {
+        let isRemoteMock
+        let savedEnvironment
+
+        beforeEach(() => {
+            isRemoteMock = jest.spyOn(ssrServerUtils, 'isRemote').mockImplementation(() => true)
+            savedEnvironment = Object.assign({}, process.env)
+            Object.assign(process.env, {
+                BUNDLE_ID: 1,
+                DEPLOY_TARGET: 1,
+                EXTERNAL_DOMAIN_NAME: 'http://www.example.com',
+                MOBIFY_PROPERTY_ID: 'example'
+            })
+        })
+
+        afterEach(() => {
+            isRemoteMock.mockRestore()
+            process.env = savedEnvironment
+        })
+
+        test('should not proxy', () => {
+            const app = createApp(opts())
+            return request(app)
+                .get('/mobify/proxy/base/test/path')
+                .expect(501)
+        })
+    })
+
```

Comment:
Originally, I wrote this test first, before I found the other existing proxying test that I changed above: 

https://github.com/SalesforceCommerceCloud/pwa-kit/pull/205/files#diff-18b9fcca6898f894e8d9db15ed9e2d38046d35f4857accb977fd1e806a987d60R198-R199

I don't feel strongly about it, but if we feel these tests are exercising the same thing, this test could be removed.